pennylane_cirq/cirq_device.py

```diff
@@ -139,20 +139,26 @@ def __init__(self, wires, shots, analytic, qubits=None):
         "CZ": CirqOperation(lambda: cirq.CZ),
         "PhaseShift": CirqOperation(lambda phi: cirq.ZPowGate(exponent=phi / np.pi)),
         "CPhase": CirqOperation(lambda phi: cirq.CZPowGate(exponent=phi / np.pi)),
-        "RX": CirqOperation(cirq.rx),
-        "RY": CirqOperation(cirq.ry),
-        "RZ": CirqOperation(cirq.rz),
-        "Rot": CirqOperation(lambda a, b, c: [cirq.rz(a), cirq.ry(b), cirq.rz(c)]),
-        "CRX": CirqOperation(lambda phi: cirq.ControlledGate(cirq.rx(phi))),
-        "CRY": CirqOperation(lambda phi: cirq.ControlledGate(cirq.ry(phi))),
-        "CRZ": CirqOperation(lambda phi: cirq.ControlledGate(cirq.rz(phi))),
-        "CRot": CirqOperation(
+        "RX": CirqOperation(lambda phi: cirq.XPowGate(exponent=phi / np.pi, global_shift=-0.5)),
+        "RY": CirqOperation(lambda phi: cirq.YPowGate(exponent=phi / np.pi, global_shift=-0.5)),
+        "RZ": CirqOperation(lambda phi: cirq.ZPowGate(exponent=phi / np.pi, global_shift=-0.5)),
+        "Rot": CirqOperation(
             lambda a, b, c: [
-                cirq.ControlledGate(cirq.rz(a)),
-                cirq.ControlledGate(cirq.ry(b)),
-                cirq.ControlledGate(cirq.rz(c)),
+                cirq.ZPowGate(exponent=a / np.pi, global_shift=-0.5),
+                cirq.YPowGate(exponent=b / np.pi, global_shift=-0.5),
+                cirq.ZPowGate(exponent=c / np.pi, global_shift=-0.5),
             ]
         ),
+        "CRX": CirqOperation(lambda phi: cirq.CNotPowGate(exponent=phi / np.pi, global_shift=-0.5)),
+        # "CRY": CirqOperation(lambda phi: cirq.ControlledGate(cirq.YPowGate(exponent=phi / np.pi, global_shift=-0.5))),
+        "CRZ": CirqOperation(lambda phi: cirq.CZPowGate(exponent=phi / np.pi, global_shift=-0.5)),
+        # "CRot": CirqOperation(
+        # lambda a, b, c: [
+        # cirq.ControlledGate(cirq.rz(a)),
+        # cirq.ControlledGate(cirq.ry(b)),
+        # cirq.ControlledGate(cirq.rz(c)),
+        # ]
+        # ),
```

Comment:
Let's keep these operations enabled for the non-floq usage.

Reply:
Okay. I thought when using them with floq, they'd then just be decomposed. Optimally, one would just work on the floq device, I guess?